images/airflow/2.9.2/python/mwaa/config/airflow.py

```diff
@@ -272,6 +277,11 @@ def _get_essential_airflow_api_config() -> Dict[str, str]:
     if os.environ.get("MWAA__CORE__AUTH_TYPE", "").lower() == "none":
         api_config["AIRFLOW__API__AUTH_BACKENDS"] = "airflow.api.auth.backend.default"
 
+    if os.environ.get("MWAA__CORE__DISABLE_CORS", "false").lower() == "true":
+        api_config["AIRFLOW__API__ACCESS_CONTROL_ALLOW_HEADERS"] = "*"
+        api_config["AIRFLOW__API__ACCESS_CONTROL_ALLOW_METHODS"] = "*"
+        api_config["AIRFLOW__API__ACCESS_CONTROL_ALLOW_ORIGINS"] = "*"
```

Comment:
I agree with @Kytha, this would open the environment up to risks.  Is the domain of requests known when the container is launched?

Reply:
Yes, it would, if set without thought. Usually when a user needs to do this they have some external authentication mechanism that is insulating their airflow environment from CSRF attacks. For that reason this config defaults to False.

Reply:
I don't think the MWAA image has any value it will enforce on these config fields. To me, it would seem more appropriate to provide these access control related configuration options as environment variables when orchestrating your containers for your use-case, instead of introducing into the image itself like this. 

This way, the image can remain un-opinionated on the access control config.